content/en/blog/_posts/2019-05-08-Kyma-extend-and-build-on-kubernetes-with-ease.md

```diff
@@ -0,0 +1,127 @@
+
+### Do not rewrite your Monoliths
+
+Rewriting is hard, costs a fortune and in majority of cases is not needed. At the end what you need is to be able to write and put on production new features quicker. You can do it by connecting your monolith into Kyma using the [Application Connector](https://kyma-project.io/docs/components/application-connector). In short, this component makes sure that securely:
+
+- You can call back registered monolith without a need of taking care of authorization, as Application Connector handles that
+- Events sent from your monolith get into Kyma's event bus
+
+Your monolith can at the moment register three different type of services: REST (with [OpenAPI](https://www.openapis.org/) specification) and OData (with Entity Data Model specification) for synchronous communication, and for asynchronous communication you can register a catalog of events based on [AsyncAPI](https://www.asyncapi.com/) specification. Your events are later delivered internally using [NATS Streaming](https://nats.io/) channel with [Knative eventing](https://github.com/knative/eventing/).
+
+Once your monolith's services are connected you can provision them in selected namespace thanks to the, previously mentioned, [Service Catalog](https://kyma-project.io/docs/components/service-catalog/) integration. So you can imagine, you as a developer go to the catalog and see a list of all the services you can consume. There are services from your monolith, and services from other 3rd party providers thanks to registered Service Brokers, like [Azure's OSBA](https://github.com/Azure/open-service-broker-azure). It is like entering a shop, a home improvement retailer, just one. You do not go for a hammer to Azure, and to GCP for screwdriver, and for wooden board to your Monolith. It is one single place with all of it. You want to build veranda in your home. You just pick up the tools from the shelves and take them. You do not write them from scratch, you have them all in one place, in one single tool belt and you focus just one one thing only, business logic, that is it.
+
+### Finally some code
+
+Below you have a sample code I had to write to integrate one Monolith with Azure services. I wanted to understand sentiments shared by customers under products review section. On every event with a review comment I wanted to use some machine learning to call some sentiments analysis service and then in case of negative comment, I wanted to persist it in some database for later review. This is a code of a function created thanks to our [Serverless](https://kyma-project.io/docs/components/serverless) component. Pay attention to my code comments: 
```

Comment:
```suggestion
Check out some code I had to write to integrate a monolith with Azure services. I wanted to understand the sentiments shared by customers under the product's review section. On every event with a review comment, I wanted to use machine learning to call a sentiments analysis service and in case of negative comment, I wanted to store it in a database for later review. This is the code of a function created thanks to our [Serverless](https://kyma-project.io/docs/components/serverless) component. Pay attention to my code comments: 
```